bin/cli.js

```diff
@@ -59,6 +63,7 @@ const getConfig = args => {
   const config = {
     header: cfg.header || args.header || '',
     footer: cfg.footer || args.footer || '',
+    removeInterface: cfg.removeInterface || args.removeInterface || false,
```

Comment:
args are supposed to overwrite what's written in the config, aren't they?
Though that's subject for another PR.